super_calendar/tests/test_super_calendar.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Odoo, Open Source Management Solution
+#
+#    Copyright (c) All rights reserved:
+#        (c) 2012      Agile Business Group sagl (<http://www.agilebg.com>)
+#        (c) 2012      Domsense srl (<http://www.domsense.com>)
+#        (c) 2015      Anubía, soluciones en la nube,SL (http://www.anubia.es)
+#                      Alejandro Santana <alejandrosantana@anubia.es>
+#        (c) 2015      Savoir-faire Linux <http://www.savoirfairelinux.com>)
+#                      Agathe Mollé <agathe.molle@savoirfairelinux.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see http://www.gnu.org/licenses
+#
+##############################################################################
+
+from openerp.tests import TransactionCase
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT, \
+    DEFAULT_SERVER_DATE_FORMAT
+
+
+class TestSuperCalendar(TransactionCase):
+
+    def setUp(self):
+        super(TestSuperCalendar, self).setUp()
+
+        self.PartnerObj = self.env['res.partner']
+        self.SuperCalendarObj = self.env['super.calendar']
+        self.SuperCalendarConfiguratorObj = self.env[
+            'super.calendar.configurator']
+        self.SuperCalendarConfiguratorLineObj = self.env[
+            'super.calendar.configurator.line']
+        self.ModelFieldsObj = self.env['ir.model.fields']
+        self.ModelObj = self.env['ir.model']
+
+        self.partner_A = self.PartnerObj.create({
+            'name': 'Partner A',
+            'date': (datetime.today() + relativedelta(days=3)),
+            'email': 'Partner A email'
+        })
```

Comment:
`psycopg2.IntegrityError: null value in column "notify_email" violates not-null constraint`